### PR TITLE
feat(websites): Add AniChart presence

### PR DESCRIPTION
**Pull request by KurokuTetsuya**

### CODE REVIEW

**Comment by KurokuTetsuya** [comment]
Proof it works:
image

File: websites/A/AniChart/presence.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const presence = new Presence({
+  clientId: "795125406264066099"
+}),
+presenceData: PresenceData = {
+  largeImageKey: "anichart"
+};
+
+presence.on("UpdateData", async () => {
+  const startTimestamp = Date.now();
+  presenceData.startTimestamp = startTimestamp;
```
Comment (ririxidev) [inline websites/A/AniChart/presence.ts:10]: browsing doesn't work like that. use
```javascript
Math.floor(Date.now() / 1000);
```
and just add it to presenceData object

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
show some more data

**Comment by KurokuTetsuya** [comment]
> show some more data

wdym?

**Comment by KurokuTetsuya** [comment]
If you're askin for more data of the website. Well, the website just showing the ongoing anime/completed anime. If you want to see the details, you'll be redirected to `anilist.co`

**Review by Bas950 — APPROVED** [review]

**Comment by Bas950** [comment]
> If you're askin for more data of the website. Well, the website just showing the ongoing anime/completed anime. If you want to see the details, you'll be redirected to `anilist.co`

my bad, I for some reason thought you could watch anime on the website too.

File: websites/A/AniChart/presence.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const presence = new Presence({
+  clientId: "795125406264066099"
+}),
+presenceData: PresenceData = {
+  largeImageKey: "anichart"
+};
+
+presence.on("UpdateData", async () => {
+  presenceData.startTimestamp = Math.floor(Date.now() / 1000);
```
Comment (ririxidev) [inline websites/A/AniChart/presence.ts:9]: i said to assign this startTimestamp to presenceData object. Or your startTimestamp msut change on every subpage?

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/A/AniChart/presence.ts (hunk 2 above)
Comment (KurokuTetsuya) [reply websites/A/AniChart/presence.ts:9]: oh lmao ok

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**